changelog/changelog.py

```diff
@@ -307,15 +316,17 @@ def main():
         'generator': default_changelog_generator,
```

Comment:
Everything before Line 300 are just format change due to PyCharm's autoformatter (plus one typo change on Line 28). 

Most other PR relevant changes are below.